.github/dependabot.yml

```
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    open-pull-requests-limit: 2
+    directory: "/"
+    schedule:
+      interval: "daily"
+    reviewers:
+      - "woocommerce/android-developers"
+    ignore:
+      - dependency-name: "com.android.tools.build:gradle"
```

AGP is a dependency we'd like to have in sync with other in-house libraries due to compatibility with composite builds. `Dependabot` will ignore any updates for that.

---

First of all: wise idea! 🙇‍♂️ 

Second: the comment in the PR would be useful in the configuration file, too. What do you think of:

```suggestion
	  # The Android Gradle Plugin is a dependency we'd like to have in sync with other
	  # in-house libraries due to compatibility with composite build.
      - dependency-name: "com.android.tools.build:gradle"
```

I expanded AGP, which might be unnecessary given this file will be touched by mostly Android developers. Still, it took me a few moments to understand what AGP meant so at least one developer, _future me_, would benefit from this longer comment 😄

---

Sure thing, good suggestion! Yeah AGP is probably not a very common shortcut, sorry for confusion 😅 addressed in fe9503f018c31b6fff7b372971fa244d153bc9a4

---

🙇‍♂️ thanks!

---

>Still, it took me a few moments to understand what AGP meant so at least one developer

Make it two 😆  (took me a few seconds before I figured that AGP was probably an acronym for Gradle then Googled it to confirm the full name of that tool and get the abbreviation 😜 )